assets/js/modules/thank-with-google/datastore/publications.js

```diff
@@ -155,6 +162,39 @@ const baseSelectors = {
 			publications[ 0 ]
 		);
 	} ),
+
+	/**
+	 * Gets the link to create new publication.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {string} Create publication URL.
+	 */
+	getServiceCreatePublicationURL: createRegistrySelector(
+		( select ) => () => {
+			const homeURL = select( CORE_SITE ).getHomeURL();
+
+			const url = new URL( 'https://publishercenter.google.com/' );
+			url.searchParams.set( 'sk_url', encodeURIComponent( homeURL ) );
```

Comment:
For consistency reason, we should use the [`addQueryArgs`](https://github.com/WordPress/gutenberg/tree/trunk/packages/url#addqueryargs) function instead of dealing with the `URL` class directly.

Reply:
Addressed this, thank you!